mon-pix/app/routes/competence-details.js

```diff
@@ -0,0 +1,24 @@
+import { inject as service } from '@ember/service';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+import Route from '@ember/routing/route';
+
+export default Route.extend(AuthenticatedRouteMixin, {
+
+  session: service(),
+
+  async model(params) {
+    const user = await this.store.findRecord('user', this.get('session.data.authenticated.userId'));
+
+    if (user.get('organizations.length') > 0) {
```

Comment:
Je comprend la protection. Il y a plusieurs endroit où on ne l'a pas mise.
Est-ce qu'on ne sur-protège pas ?
Aucun problème à laisser, mais le retirer ne poserait pas trop de problème aussi je pense.

Reply:
Ce sera obsolete de toutes façons lorsque les routes seront ainsi:
this.route('comptentes', () => {
  index --> profilv2
  /:id --> competence-details
})
alors le blockage se fera sur la route 'competences'

Reply:
Non bloquant donc :)